concourse-resources/pipeline.yml

```diff
@@ -83,21 +62,22 @@ jobs:
               - -cxe
               - |
                 touch tag-files/additional tag-files/current
+                this_tag=$(git --git-dir source-code-tagged/.git describe --tags --abbrev=0)
+
+                git --git-dir source-code-tagged/.git checkout master
                 latest_tag=$(git --git-dir source-code-tagged/.git describe --tags --abbrev=0)
-                this_tag=$(git --git-dir source-code-tagged/.git for-each-ref refs/tags --sort=-taggerdate --format='%(refname:short)' --count=1)
+                git --git-dir source-code-tagged/.git checkout "$this_tag"
+
                 echo "$this_tag" > tag-files/current
-                [[ "$this_tag" = "$latest_tag" ]] && echo 'latest' > tag-files/additional
+                if [ "$this_tag" = "$latest_tag" ]; then
+                  echo 'latest' > tag-files/additional
+                else
+                  echo 'not latest'
+                fi
       - put: docker-hub-image
         params:
           additional_tags: tag-files/additional
           build: source-code-tagged
           tag_file: tag-files/current
           tag_as_latest: false
-    on_success:
-      put: notify
-      params:
-        text: ":white_check_mark: `tagged` output-producer-resource docker image successfully uploaded to dockerhub."
-    on_failure:
-      put: notify
-      params:
-        text: ":warning: There was a problem building/uploading `tagged` output-producer-resource docker image to dockerhub."
+          get_params: {skip_download: true}
```

Comment:
Can / should we add this skipping of the image pull to the `build-and-publish-image` job as well?